resources/web/docs_js/index.js

```diff
@@ -128,14 +128,19 @@ function init_toc(lang_strings) {
   var v_selected = title.find('select option:selected');
   title
     .find('select')
-    .change(function() {
-       var version = title.find('option:selected').val();
-       utils.get_current_page_in_version(version).fail(function() {
-         v_selected.attr('selected', 'selected');
-         alert(lang_strings('This page is not available in the docs for version:')
-               + version);
-       });
-     });
+    .change(function(e) {
+      var version = $(e.target).find('option:selected').val();
+      if (version === "other") {
+        $("#other_versions").show();
+        $("#live_versions").hide();
```

Comment:
Should this toggle the visibility in the other direction if `version !== "other"`? If so, using [`.toggle()`](https://api.jquery.com/toggle-event/) might be a more inclusive implementation.

Reply:
Once you pick "other" you can get all the versions so I don't think we need to toggle.